### PR TITLE
meteor lake ec gpio macro generation script

### PR DESCRIPTION
**Pull request by philipandag**

### CODE REVIEW

File: scripts/mtl_ecspy_to_c.py
Comment (mkopec) [inline scripts/mtl_ecspy_to_c.py:1]: I don't think this is MTL specific, you can rename to simply `ecspy_to_c.py`

Comment (philipandag) [reply scripts/mtl_ecspy_to_c.py:1]: renamed: [7a55956](https://github.com/Dasharo/ec/pull/61/commits/7a55956472a497464312cc388e654cb039558450)

**Comment by mkopec** [comment]
The log format for GPIO group M is a bit different, right now the script doesn't output anything for those pins. Please also add support for group M.

File: scripts/ecspy_to_c.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+import sys
+import re
+
+
+gpcr_control_pin_mode_to_c_define = {
+    "00": "GPIO_ALT1",
```
Comment (mkopec) [inline scripts/ecspy_to_c.py:7]:
```suggestion
    "00": "GPIO_ALT",
```

**Review by mkopec — APPROVED** [review]

**Pull request merged**